### ISSUE TITLE
sozu dumped core

### ISSUE THREAD

**Issue by Keruspe** [sozu-proxy/sozu#244]
```
#0  0x00007fef4c631536 in strlen () from /usr/x86_64-pc-linux-gnu/lib/libc.so.6
#1  0x000055e7c82fef12 in std::ffi::c_str::CStr::from_ptr () at /checkout/src/libstd/ffi/c_str.rs:749
#2  std::sys_common::gnu::libbacktrace::resolve_symname<closure> () at /checkout/src/libstd/sys_common/gnu/libbacktrace.rs:80
#3  std::sys_common::backtrace::filter_frames::{{closure}} () at /checkout/src/libstd/sys_common/backtrace.rs:110
#4  core::slice::{{impl}}::position::{{closure}}<std::sys_common::backtrace::Frame,closure> () at /checkout/src/libcore/slice/mod.rs:1206
#5  0x000055e7c830aaf1 in core::slice::Iter<std::sys_common::backtrace::Frame>::search_while<std::sys_common::backtrace::Frame,core::option::Option<usize>,closure> () at /checkout/src/libcore/slice/mod.rs:1273
#6  core::slice::{{impl}}::position<std::sys_common::backtrace::Frame,closure> () at /checkout/src/libcore/slice/mod.rs:1205
#7  std::sys_common::backtrace::filter_frames () at /checkout/src/libstd/sys_common/backtrace.rs:108
#8  std::sys_common::backtrace::_print () at /checkout/src/libstd/sys_common/backtrace.rs:73
#9  0x000055e7c8313554 in std::sys_common::backtrace::print () at /checkout/src/libstd/sys_common/backtrace.rs:60
#10 std::panicking::default_hook::{{closure}} () at /checkout/src/libstd/panicking.rs:381
#11 0x000055e7c83132c3 in std::panicking::default_hook () at /checkout/src/libstd/panicking.rs:397
#12 0x000055e7c8313a18 in std::panicking::rust_panic_with_hook () at /checkout/src/libstd/panicking.rs:577
#13 0x000055e7c8313875 in std::panicking::begin_panic<alloc::string::String> () at /checkout/src/libstd/panicking.rs:538
#14 0x000055e7c83137ea in std::panicking::begin_panic_fmt () at /checkout/src/libstd/panicking.rs:522
#15 0x000055e7c831377b in std::panicking::rust_begin_panic () at /checkout/src/libstd/panicking.rs:498
#16 0x000055e7c834c6c1 in core::panicking::panic_fmt () at /checkout/src/libcore/panicking.rs:71
#17 0x000055e7c834c72e in core::option::expect_failed () at /checkout/src/libcore/option.rs:839
#18 0x000055e7c80fc89c in sozu::command::CommandServer::handle_worker_events::h9f6f3f7b80c70716 ()
#19 0x000055e7c80f82c7 in sozu::command::CommandServer::run::h795169c6e3fca0f0 ()
#20 0x000055e7c8102539 in sozu::command::start::hf7a3648d86cbd545 ()
#21 0x000055e7c8111069 in sozu::main::h1b79d874f10c371d ()
#22 0x000055e7c831aafd in panic_unwind::__rust_maybe_catch_panic () at /checkout/src/libpanic_unwind/lib.rs:99
#23 0x000055e7c831413c in std::panicking::try<(),closure> () at /checkout/src/libstd/panicking.rs:459
#24 std::panic::catch_unwind<closure,()> () at /checkout/src/libstd/panic.rs:361
#25 std::rt::lang_start () at /checkout/src/libstd/rt.rs:59
#26 0x00007fef4c5cf540 in __libc_start_main () from /usr/x86_64-pc-linux-gnu/lib/libc.so.6
#27 0x000055e7c800604a in _start ()
```

**Comment by Geal** [comment]
what was sozu doing at the time?

**Comment by Geal** [comment]
Might be fixed by 0b6e785

**Comment by Geal** [comment]
apparently solved
